linera-base/src/data_types.rs

```diff
@@ -284,6 +284,8 @@ pub struct Resources {
     pub message_size: u32,
     /// An increase in the amount of storage space.
     pub storage_size_delta: u32,
+    /// A number of HTTP requests to be performed.
+    pub http_requests: u32,
```

Comment:
Why do we need to track this?

Reply:
This type is used to calculate grants. Basically you configure an instance of this with whichever operations you expect to be executed when the message is received, and then the runtime calculates how much that would cost.